wallet/src/androidTest/java/com/android/identity_credential/wallet/ui/TimeAgoTest.kt

```diff
@@ -0,0 +1,145 @@
+package com.android.identity_credential.wallet.ui
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.runner.RunWith
+import androidx.compose.ui.test.junit4.createComposeRule
+import com.android.identity_credential.wallet.WalletApplication
+import kotlinx.datetime.Instant
+import org.junit.Assert
+import org.junit.Test
+
+@RunWith(AndroidJUnit4::class)
+class TimeAgoTest {
```

Comment:
I think this is required for espresso tests (and, unfortunately, regular Java test will not work here because we need resources and because this finction is marked Compose).